src/drtsans/tof/eqsans/elastic_reference_normalization.py

```diff
@@ -58,16 +58,16 @@ def reshape_q_wavelength_matrix(i_of_q: IQmod) -> tuple:
 
     """
     # Create a matrix for q, wavelength, intensity and error
-    if i_of_q.delta_mod_q is None:
-        i_q_wl_matrix = np.array([i_of_q.mod_q, i_of_q.wavelength, i_of_q.intensity, i_of_q.error])
+    if i_of_q.delta_x is None:
```

Comment:
are you using `delta_x` instead of `delta_mod_q` because `i_of_q` could be an `IAnnular` object?

Reply:
Yes. I asked Yingrui whether these corrections are valid for annular binning and for now we are working under the assumption that they _are_ valid.